packages/connectivity-tests/test/localhost.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I think we can use the docker repo here too, docker-compose support vars interpolation, always wanted to do that for server version and ports.

Reply:
@mcasimir How would we use it for that? We’d have to assume that nothing is running on 27017 in that case, right?

Reply:
I guess though evergreen wont be happy that we are starting something on port 27017? Is that the issue here?

Reply:
I haven’t tried that, no, but that’s the only reason this is using docker

Reply:
(and also that you can run this locally for debugging)